package.json

```diff
@@ -26,13 +25,11 @@
   },
   "homepage": "https://github.com/Groww/react-charts/blob/main/README.md",
   "scripts": {
-      "build": "yarn build:commonjs && yarn build:es && yarn build:types",
-      "build:commonjs": "rimraf ./lib && rollup -c && rollup-plugin-visualizer stats-react.json",
-      "build:es": "rimraf ./es && babel --extensions .ts,.tsx  ./src --out-dir es",
+      "build": "yarn build:commonjs && yarn build:types",
+      "build:commonjs": "rimraf ./dist && rollup -c && rollup-plugin-visualizer stats-react.json",
```

Comment:
We not using visualizer plugin in our rollup plugin, i guess we can remote it from here as well as from our dev dependencies